src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -140,4 +135,15 @@ public boolean equals(Object other) {
                 || (other instanceof UniquePersonList // instanceof handles nulls
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
+
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>() {
+            @Override
+            public int compare(Person o1, Person o2) {
+                return o1.getName().toString().compareTo(o2.getName().toString());
```

Comment:
I think maybe can implement Comparable in Name class and add a compareTo(Name other) method